upgrade-notes/backend_20240912_121631.md

```diff
@@ -0,0 +1,3 @@
+#### Removed transport type entity and replaced it with enum class ([#3431](https://github.com/shopsys/shopsys/pull/3431))
+
```

Comment:
the BC breaks should be mentioned here (but if this goes to 16.0, then it is ok :smile: )

Reply:
what exactly should I mention? 🤔 I don't know how to exactly write it 😅